docs/Installation.md

```diff
@@ -192,10 +192,10 @@ asdf plugin-add elixir
 # latest official Nerves systems are compatible with the versions below. In
 # general, differences in patch releases are harmless. Nerves detects
 # configurations that might not work at compile time.
-asdf install erlang 24.3.2
-asdf install elixir 1.13.3-otp-24
-asdf global erlang 24.3.2
-asdf global elixir 1.13.3-otp-24
+asdf install erlang 25.0.3
```

Comment:
```suggestion
asdf install erlang 25.0.4
```

Reply:
I really don't think it matters, but the systems are still on 24.0.3.

Reply:
I'm updating `nerves_system_br` to 25.0.4. PR should be up shortly...